devicetypes/qubino/qubino-flush-2-relay.src/qubino-flush-2-relay.groovy

```diff
@@ -141,6 +141,8 @@ def configure() {
 		// additionally, manual states that default value for below parameters is 0, which disables power reporting
 		cmds += encap(zwave.configurationV1.configurationSet(parameterNumber: 42, size: 2, scaledConfigurationValue: 300))
 		cmds += encap(zwave.configurationV1.configurationSet(parameterNumber: 43, size: 2, scaledConfigurationValue: 300))
+	} else if (zwaveInfo?.model?.equals("0052")) {
+		cmds += encap(zwave.configurationV1.configurationSet(parameterNumber: 40, size: 1, scaledConfigurationValue: 75))
```

Comment:
would we not expect the same behavior from the other devices on this DTH?

Reply:
Only two devices in this DTH have power consumption. There are more configuration parameters for Relay 2, but for Relay 1 I needed only to change the value for this parameter.

Reply:
Okay. Can you add a small comment explaining what this is doing here. From what I can gather it changes some threshold for reporting power?

Reply:
Fixed. Threshold for low power devices was too small and values were "jumping" all the time